src/main/java/net/fabricmc/loader/launch/knot/KnotServer.java

```diff
@@ -22,6 +22,7 @@
 
 public class KnotServer {
 	public static void main(String[] args) {
+		System.setProperty("log4j.configurationFile", "classpath:log4j2.fabric.xml");
```

Comment:
Only when the property is not set, please! Consider moving this into Minecraft game provider as this log4j config is really minecraft specific.

Reply:
Same as above, this is here for testing right now.